joint_state_publisher_gui/joint_state_publisher_gui/joint_state_publisher_gui.py

```diff
@@ -193,6 +206,15 @@ def initializeSliders(self):
         # Set zero positions read from parameters
         self.centerEvent(None)
 
+        # Set size of min size of window based on number of sliders.
+        if len(self.sliders) >= INIT_NUM_SLIDERS:  # Limits min size to show INIT_NUM_SLIDERS
+            num_sliders = INIT_NUM_SLIDERS
+        else:
+            num_sliders = len(self.sliders)
+        scroll_layout_height = num_sliders * DEFAULT_SLIDER_HEIGHT
+        scroll_layout_height += (num_sliders - 1) * DEFAULT_CHILD_MARGIN
```

Comment:
```suggestion
        scroll_layout_height += (num_sliders + 1) * DEFAULT_CHILD_MARGIN
```
I'm uncertain if there's a child margin between the top and bottom of the scroll layout, or if there's only child margins between the slider. I think this is actually correct although I can't find an explicit reference in the QT documentation.

Reply:
I did a bit of testing here, and I think it should be `+` here.  At least, with the minus in place, the scroll-bars were always there on my setup, even with just two links.

Reply:
Thanks for confirming @clalancette . I was pretty sure it should have been `+`